auth/client-api.md

```diff
@@ -14,7 +14,7 @@ abstract class Client {
   public abstract request(params: RequestParams): Promise<{ uri, id }>;
 
   // respond wallet authentication
-  public abstract respond(params: RespondParams): Promise<boolean>;
+  public abstract respond(result: RespondParams | Error): Promise<boolean>;
```

Comment:
Is `Error` structure defined somewhere? I know that's JsonRpcError but it'd be nice to mention it

Reply:
I do not think it is a jsonrpc error, it is more like an aggregation of jsonrpc error and internal signature validation error

Reply:
So it's even more required to have this structure defined somewhere

Reply:
Not necessarily a structure but I think we could agree on error codes

Reply:
I know what you mean:
```
Error {
code: number
message: string
}
```

Reply:
yeah we can add it if you guys accept the PR